translations/ui-invoice/en.json

```diff
@@ -15,6 +15,7 @@
   "button.remove": "Remove",
 
   "showTags": "Show tags",
+  "addTagFor": "Add tag for: ",
```

Comment:
I'm not sure if it's a good idea to include punctuation marks and whitespaces to translation labels. 
cc @dltj